app/static/src/scss/includes/_breadcrumbs.scss

```diff
@@ -63,38 +63,38 @@
   }
 
   &__link--record {
-    color: #1d70b8;
+    color: $colour-link-visited;
     font-size: 1rem;
     font-weight: 400;
     margin-top: 0;
 
     &--series {
       white-space: nowrap;
-      color: #1d70b8;
+      color: $colour-link-visited;
 
       &:visited {
-        color: #1d70b8;
-        -webkit-text-fill-color: #1d70b8;
+        color: $colour-link-visited;
+        -webkit-text-fill-color: $colour-link-visited;
       }
     }
 
     &--transferring-body {
       flex-wrap: wrap;
-      color: #1d70b8;
+      color: $colour-link-visited;
 
       &:visited {
-        color: #1d70b8;
-        -webkit-text-fill-color: #1d70b8;
+        color: $colour-link-visited;
+        -webkit-text-fill-color: $colour-link-visited;
       }
     }
 
     &--consignment {
       white-space: nowrap;
-      color: #1d70b8;
+      color: $colour-link-visited;
 
       &:visited {
```

Comment:
I see we mention visited and normal color for links quite a few times for different elements, perhaps we could have an overwrites file that can do this for us for links? Might save on quite a bit of CSS space and make these more readable